src/screens/DomainScreen/index.js

```diff
@@ -244,7 +244,7 @@ const DomainScreen = () => {
   }
   return (
     <View style={styles.container}>
-      <StatusBar translucent={false} />
+      <StatusBar translucent={false} barStyle={'light-content'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `barStyle` prop of the `StatusBar` component has been changed to `'light-content'`. This will change the color of the status bar content to light, which is suitable for dark backgrounds. Ensure that this change doesn't affect readability on screens with lighter backgrounds.
```diff
- <StatusBar translucent={false} />
+ <StatusBar translucent={false} barStyle={'light-content'} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->